tests/models/dpt/test_image_processing_dpt.py

```diff
@@ -138,3 +168,124 @@ def test_keep_aspect_ratio(self):
         pixel_values = image_processor(image, return_tensors="pt").pixel_values
 
         self.assertEqual(list(pixel_values.shape), [1, 3, 512, 672])
+
+    def test_call_segmentation_maps(self):
+        # Initialize image_processing
+        image_processing = self.image_processing_class(**self.image_processor_dict)
```

Comment:
nit, `image_processor` would be better

Reply:
Renamed `image_processing` to `image_processor`. Should I also rename it in the other tests?